Tests/Test_Amperes_Task.c

```diff
@@ -32,8 +32,6 @@
  *    verify that the contactor opens (the contactor should have closed earlier on startup)
  *****************************************************************************/
 
-void EnterFaultState(void);
```

Comment:
Since we are getting rid of the faultstate task, we need to verify that the enterfaultstate() method works in the program.

Reply:
Or just include Tasks.h

Reply:
Wouldn't it work if it compiles?